src/LogoFX.Client.Core.Platform/winui3/LogoFX.Client.Core.Platform.WinUI3.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <RootNamespace>LogoFX.Client.Core.Platform.WinUI3</RootNamespace>
```

Comment:
@glyad fix the namespace - see other platform projects for reference

Reply:
Fixed